### ISSUE TITLE
Consider messing with TCP_QUICKACK.

### ISSUE THREAD

**Issue by lmwnshn** [cmu-db/noisepage#1178]
TL;DR: Leave TCP_QUICKACK enabled (set by default if unspecified), and leave TCP_NODELAY enabled (set).

Just documenting a conversation with @gonzalezjo so that it doesn't get lost on Slack. Edited for brevity. 

**W**: Should we consider messing with TCP_QUICKACK?
**J**: Delayed ACKs are an optimization for the kind of work that we're doing. I'll benchmark it, but I would expect a performance regression from what we have [if you switch it off].
**W**: Right now, TCP_QUICKACK is not specified by the server when we're creating sockets. This means that it is on by default. Apparently Linux does a thing where it is on by default and switches to off if it detects you're doing HTTP or some magic. From unscientific running on my desktop.
**J**: Just ran a bunch of benchmarks and that matches what I've noticed. Absolutely no clear difference with or without QUICKACK. If the kernel is switching it on/off automatically, then that would explain it. Tried QUICKACK and NODELAY, each separately, and neither. Doesn't seem possible to improve on the current configuration in master.
